diff --git a/tests/utils.py b/tests/utils.py
--- a/tests/utils.py
+++ b/tests/utils.py
@@ -870,7 +870,10 @@ def fork_new_process_for_each_test(func: Callable[_P, None]) -> Callable[_P, No
     def wrapper(*args: _P.args, **kwargs: _P.kwargs) -> None:
         # Make the process the leader of its own process group
         # to avoid sending SIGTERM to the parent process
-        os.setpgrp()
+        try:
+            os.setpgrp()
+        except PermissionError:
+            pass
         from _pytest.outcomes import Skipped
